mongo/client.go

```diff
@@ -180,9 +193,10 @@ func (c *Client) Disconnect(ctx context.Context) error {
 	return nil
 }
 
-// Ping verifies that the client can connect to the topology.
-// If readPreference is nil then will use the client's default read
-// preference.
+// Ping sends a ping command to verify that the client can connect to the cluster.
```

Comment:
Maybe discuss what happens if it's not up -- e.g. serverSelectionTimeout delay and server selection error.  I think it's important to keep reminding users of our connectivity model.